src/icalendar/prop.py

```diff
@@ -250,7 +252,7 @@ def __init__(self, dt_list):
         self.dts = vDDD
 
     def to_ical(self):
-        dts_ical = (dt.to_ical() for dt in self.dts)
+        dts_ical = (from_unicode(dt.to_ical()) for dt in self.dts)
```

Comment:
Hi, I do not block a merge, just want to have some conversation: How did you discover that this is helpful? What problem does it solve?

Reply:
I downloaded the crashing testsuite provided by OSS-Fuzz and passed it into the fuzzer as its input. Then, discovered that the testcase was crashing in this function due to attempting to join a string as bytes.

Reply:
Oh, that is interesting. Is that something that we can add to the project as tests that other people can run, too?

Reply:
Definitely, we can migrate the corpus from OSSFuzz to be hostedi n the `fuzzing` directory